src/app/state_controller.cpp

```diff
@@ -228,3 +264,95 @@ Application_State State_Controller::operator()(Find_Account& state)
     }
     return Find_Account { "Invalid ID" };
 }
+
```

Comment:
I'm not sure what's the purpose of the Get_Type, Get_First etc. states. Since they all do the same thing just run a loop in the operator(Create_Account&) function

Reply:
those states no longer exist